apps/condo/domains/property/components/panels/Builder/forms/UnitForm.tsx

```diff
@@ -0,0 +1,235 @@
+/** @jsx jsx */
```

Comment:
The same question as about `SectionForm` and `ParkingForm`.
Can we make some base forms?

Reply:
Yeah, I think there could be done with something like "adapter" class. I try to figure it out later